libuplink-android/src/main/java/io/storj/EncryptionAccess.java

```diff
@@ -51,6 +51,22 @@ public void merge(EncryptionAccess other) throws StorjException {
         }
     }
 
+    /**
+     * Overrides the encryption key for the prefix with encryption key.
+     *
+     * @param bucket bucket name
+     * @param path   path in bucket
+     * @param key    the key
+     * @throws StorjException in case of error
+     */
+    public void overrideEncryptionKey(String bucket, String path, byte[] key) throws StorjException {
```

Comment:
How about renaming the `path` parameter to `prefix`? This is what we are going to use as name in Go uplink 1.1.

Reply:
sure, I missed that